packages/ui/RadioDecorator/tests/RadioDecorator-specs.js

```diff
@@ -13,51 +14,53 @@ describe('RadioDecorator', () => {
 
 	const Controller = RadioControllerDecorator('main');
 
+	const expectToBeActive = (controller, decorator) => {
+		expect(controller.active).toBe(decorator && decorator.handleDeactivate);
+	};
+
 	test('should be activated when its prop is true on mount', () => {
+		const controllerRef = createRef();
```

Comment:
Oh, why did you change these? Please don't make changes tests were working. :)
The comment on Jira was confined to the skipped test.